pom.xml

```diff
@@ -200,7 +200,13 @@ Brotli, Zstandard and ar, cpio, jar, tar, zip, dump, 7z, arj.
       <artifactId>slf4j-api</artifactId>
       <version>${slf4j.version}</version>
       <scope>test</scope>
-    </dependency>    
+    </dependency>
```

Comment:
Remove this dep unless required.

Reply:
Not required, but it's only in test scope and just so we can see any logs produced by the application while running the tests. Without this, they get swallowed.